spring-cloud-aws-samples/spring-cloud-aws-parameter-store-sample/pom.xml

```diff
@@ -17,6 +17,12 @@
 			<artifactId>spring-boot-starter</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-bootstrap</artifactId>
```

Comment:
Is this dependency really needed for a sample?

Reply:
not at all!